frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/$childId/federal-provincial-territorial-benefits.tsx

```diff
@@ -379,7 +379,7 @@ export default function AccessToDentalInsuranceQuestion() {
               </Button>
               <ButtonLink
                 id="back-button"
-                routeId="$lang+/_public+/apply+/$id+/adult-child/review-adult-information"
+                routeId="$lang+/_public+/apply+/$id+/adult-child/children/index"
```

Comment:
It's the `cancel` button when in edit mode. it should be `"$lang+/_public+/apply+/$id+/adult-child/review-adult-information"`.